_sass/base.scss

```diff
@@ -5,7 +5,9 @@ html, body {
   width: 100%;
```

Comment:
I think the width has enough heart and grit to pull 110% of the weight instead of just 100% here.